hw/ip/aes/doc/_index.md

```diff
@@ -133,24 +139,48 @@ The AES unit operates as follows:
 1. The initial key and configuration is provided to the AES unit via a set of control and status registers (CSRs) accessible by the processor via TL-UL bus interface.
    Each key register has to be written at least once.
    The order in which the registers are written does not matter.
+1. The processor provides the initialization vector (IV) or initial counter value to the four IV registers via TL-UL bus interface in CBC or CTR mode, respectively.
+   Each IV register has to be written at least once.
```

Comment:
"has to" -> "must" ?  ("shall"?)  Depends on how specification/requirement-like you want it to sound.

Reply:
I prefer not to use "shall". Changed all occurrences of "has to" to "must".